### PR TITLE
Paying service

### PR DESCRIPTION
**Pull request by AnastasiaBerlina**

### CODE REVIEW

**Comment by AnastasiaBerlina** [comment]
[[FE+BE] payment servce](https://trello.com/c/lqWeXDzg/112-febe-payment-servce)

**Comment by AnastasiaBerlina** [comment]
[[BE] purchase confirmation](https://trello.com/c/qVDGOZDU/105-be-purchase-confirmation)

**Comment by AnastasiaBerlina** [comment]
[[BE] purchase confirmation](https://trello.com/c/qVDGOZDU/105-be-purchase-confirmation)

File: src/main/java/com/epam/lowcost/model/User.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     private String firstName;
     private String lastName;
     private String documentInfo;
-    private LocalDateTime birthday;
+    private LocalDate birthday;
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/model/User.java]: why u change type?

Comment (AnastasiaBerlina) [reply src/main/java/com/epam/lowcost/model/User.java]: why do we need  user's time of birth?

**Comment by EugeneGrim** [comment]
image
Fix flight search when buying a ticket

**Comment by EugeneGrim** [comment]
Resolve merge conflicts

**Pull request merged**